### ISSUE TITLE
MSUnmerged - Filter RSEs which do not support nonempty directory removal through WebDAV  

### ISSUE THREAD

**Issue by todor-ivanov** [dmwm/WMCore#11839]
**Impact of the bug**
MSUnmerged

**Describe the bug**
In order to avoid blocking the cleaning of all regular RSEs which support nonempty directory removal through WebDav protocol, we want to filter them from the MSUnmerged instances that we already have, as defined in the Kuberenetes  repository:
https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/services/reqmgr2ms-unmerged-t1.yaml
https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/services/reqmgr2ms-unmerged-t2t3.yaml
https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/services/reqmgr2ms-unmerged-t2t3us.yaml

We can then try to put them in a separate pod on a later stage.  So far they are not succeeding in completing the cleanup cycles. We have a short short summary of the status of all the sites from earlier this month: [1], while the current status of all RSEs can be seen at [2].  And the results so far show a handful of RSEs that need to be filtered  at this stage, in order not to block the rest:
* `T2_US_Wisconsin`
* `T2_US_Caltech`
* `T2_CH_CERN` - Which will need a special attention.

**How to reproduce it**
Non Reproducible 

**Expected behavior**
All RSEs to complte their cleanup cycles.

**Additional context and error message**
[1]
[allRses-2023-12-14.txt](https://github.com/dmwm/WMCore/files/13753372/allRses-2023-12-14.txt)

[2]
https://cmsweb.cern.ch/ms-unmerged/data/info?rse=ALL_DOCS

**Comment by todor-ivanov** [comment]
hi @amaltaro ,

Please take a look at those 3 merge requests in the `services_config` repository [1]. If you agree on the approach, please let me know, I will merge and redeploy the services in production, such that we reassure at least those RSEs which can run uninterrupted during the holidays. For the filtered  ones we will have to create a completely new deployment schema. One possibility would be what we discussed during the Monday meeting - a separate pod for each of them. But upon thinking twice on this, I am now less convinced we should even create any pod for them at this stage. Because those RSEs has little to no chance for completing their cleanup cycle, being them cleaned from a separate pod or not.    


[1]
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/239
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/240
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/241

**Comment by amaltaro** [comment]
@todor-ivanov thanks for checking in. Go ahead with your plan.

**Comment by todor-ivanov** [comment]
hi @amaltaro all the pods in production have been reconfigured and running with the new config:
```
config.views.data.skipRSEs = ['T2_US_Caltech_Ceph', 'T2_US_Caltech', 'T2_US_Wisconsin', 'T2_CH_CERN']
```

I just checked the logs and the RSEs in the skip list are indeed not iterated  through:
```
2024-01-01 17:14:53,920:INFO:MSUnmerged: plineUnmerged: Run on RSE: T2_US_Wisconsin is skipped due to a restriction set in msConfig. Will NOT retry until the RSE is removed from 'skipRSEs' list.
```

Closing this issue with the current comment. 

p.s. 
 Currently , there is  problem related to the empty list of scanned RSEs at RucioConMon [1] but for that I am going to ping dataops and let them fix on their own convenience.  

[1]
https://cmsweb.cern.ch/rucioconmon/unmerged/index
